cdk/kraken/src/deploy.ts

```diff
@@ -73,6 +73,11 @@ export class DeployJob extends CheckoutJob {
           {
             id: "synth",
             name: "Synth cdk8s manifests",
+            ...(fullConfig.isFeatureDeploy
```

Comment:
Consider moving the check for PR number earlier to skip feature branch deploy job as a whole